app/views/group/index.html.erb

```diff
@@ -0,0 +1,22 @@
+  <div class="group-container">
+    <h1 class="group_title">Categories</h1>
+    <% @groups.each do |group| %>
+    <div class="group">
+    <div class="amount-name">
+    <h2><%= link_to group.name, group_path(group), class: 'group_link'%></h2>
+          <p class="group_amount">
+            $<%= group.items.sum(:amount).to_f %>
+          </p>
+    </div>
+      <div class="group_item">
+        <p class="group_p"><%= "#{group.icon}" %></p>
+          <p class="group_date"><%= group.created_at.strftime("%B %d, %Y") %></p>
+          </div>
+    </div>
+    <% end %>
+  </div>
+  <div class="add_new_container">
+<div class="add_new">
+ <%= link_to 'Add New Category', new_group_path, class: 'add_link' %>
```

Comment:
- Add padding around this text such that it can look like the original design.
### Your implementation
![image](https://github.com/david-oganization/rails-capstone-project/assets/45460192/4358d68e-56b1-40b6-a864-95b09dcfffce)
### Required
![image](https://github.com/david-oganization/rails-capstone-project/assets/45460192/4b96d351-86b4-4716-8052-0d376e658808)